components/brave_wallet_ui/components/desktop/card-headers/account-details-header.tsx

```diff
@@ -192,6 +192,16 @@ export const AccountDetailsHeader = (props: Props) => {
         (option: AccountButtonOptionsObjectType) => option.id !== 'privateKey'
       )
     }
+    // We are currently not able to support viewing a
+    // BTC or ZEC account on a block explorer.
```

Comment:
Do we have an issue logged for this? If so can you include the link to it?

Reply:
Issue created https://github.com/brave/brave-browser/issues/39699 and comment updated!